setup.sh

```diff
@@ -0,0 +1,382 @@
+    echo
+    curl -XGET $composerYMLURL > $composerYMLPath
+  elif commandExists wget; then
+    echo
+    writeBold "Downloading Kuzzle launch file..."
+    echo
+    wget -O $composerYMLPath $composerYMLURL
+  else
+    writeBold "$RED" "[✖] curl or wget need to be installed to download the Kuzzle launch file,"
+    writeBold "$RED" "    but none seems to be installed on your system."
+    echo
+    writeBold "$BLUE" "Please install curl or wget and re-run this script."
+    echo
+    exit 3
+  fi
+  echo
+  writeBold "$GREEN" "[✔] The Kuzzle launch file has been successfully downloaded."
+  writeBold          "    This script can launch Kuzzle automatically or you can do it"
+  writeBold          "    manyally using Docker Compose."
+  write              "    To manyally launch Kuzzle you can type the following command:"
+  write              "    docker-compose -f $composerYMLPath up"
+  while [[ "$launchTheStack" != [yYnN] ]]
+    do
+      promptBold "[❓] Do you want to start Kuzzle now? (y/N) "
+      read launchTheStack trash
+      case "$launchTheStack" in
+        [yY])
+          echo
+          writeBold "Starting Kuzzle..."
+          $DOCKER_COMPOSE_BIN -f $composerYMLPath up -d
+          echo
+          write "$GREEN" "[✔] Kuzzle is up and running!"
+          ;;
+        [nN] | '')
+          echo
+          writeBold "$BLUE" "Ok."
+          launchTheStack=n
+          ;;
+        *)
+          echo
+          writeBold "$RED" "[✖] I did not understand your answer."
+          ;;
+      esac
+    done
+  echo
+  writeBold "Where do we go from here?"
+  shortHelp
+}
+
+shortHelp() {
+  echo
+  writeBold "* You can open this short help by calling ./setup.sh --help"
+  writeBold "* You can start Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath up -d"
+  writeBold "* You can see the logs of the Kuzzle stack by typing:"
+  write "  docker-compose -f $composerYMLPath logs -f"
+  writeBold "* You can check if everything is working by typing:"
+  write "  curl -XGET http://localhost:7512/"
+  writeBold "* You can stop Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath stop"
+  writeBold "* You can restart Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath restart"
+  writeBold "* You can read the docs at http://docs.kuzzle.io/"
+  echo
+}
+
+# Main execution routine
+# ===========================
+
+if [ "$1" == "--help" ]; then
+  echo
+  writeBold "# Kuzzle short help"
+  writeBold "  ================="
+  shortHelp
+  exit 0
+fi
+
+echo
+writeBold "# Kuzzle Setup"
+writeBold "  ============"
+echo
+writeBold "This script will help you launch Kuzzle and install"
+writeBold "all the necessary dependencies."
+echo
+write     "* You can refer to http://docs.kuzzle.io/ if you need better"
+write     "  understanding of the installation process."
+write     "* Feel free to join us on Gitter at https://gitter.im/kuzzleio/kuzzle"
+write     "  to get help in real time."
```

Comment:
I would remove the `in real time` part, as we don't exactly have a H24 support ;-)